engine/client/gl_local.h

```diff
@@ -349,6 +349,72 @@ void R_LightForPoint( const vec3_t point, color24 *ambientLight, qboolean invLig
 int R_CountSurfaceDlights( msurface_t *surf );
 int R_CountDlights( void );
 
+
+/// <STROBE> // Move to strobe.h ?
+//#define SolidConsoleX 0
+//#define SolidConsoleY (y - scr_width->value * 3 / 4)
+//#define SolidConsoleW (scr_width->value)
+//#define SolidConsoleH (scr_width->value * 3 / 4)
+//extern wrect_t con_rect; //Float - Int incompatibility
+typedef struct _rectf_s
+{
+	float x, y, w, h;
+} _rectf_t;
+extern _rectf_t con_rect;
+
+void R_Strobe( void );
+
+typedef enum {
+	p_positive = 1, // Phase: Positive
+	p_inverted = 1 << 1, // Phase: Inverted
+	f_normal = 1 << 2 // Frame: Normal
+} fstate_e; // Frame State
+
+typedef struct StrobeInfo_s {
+	unsigned int fCounter; // Frame counter
+	unsigned int pCounter, pNCounter, pBCounter; // Positive phase counters
+	unsigned int nCounter, nNCounter, nBCounter; // Negative phase counters
+	double deviation; // Standard deviation
+	double cdTimer; // Cooldown timer
+	fstate_e frameInfo; // Frame info
+} StrobeInfo_t;
+
+extern StrobeInfo_t StrobeInfo;
```

Comment:
If this defintions are not used by other parts of game engine, it's better to keep them encapsulated.

Reply:
Encapsulating by moving to separate header file dedicated to strobing? If that is the case, I agree.

Reply:
Just as you want.

It's not limited to headers. You maybe want to define them in *.c file and declare as "static", so they wouldn't accessible from other compilation units.